### PR TITLE
Fix glossary

### LINKED ISSUE #136: Fix wrong definition of tags in glossary
Tags are to provide dimension to the collected data and propagate the dimension, not to store information. 

See the section where it says:

```
Tags are key-value pairs that are used to store information about metrics for example;
```

### PR DESCRIPTION
**Pull request by rakyll**
Various fixes to improve misleading/inaccurate definitions.

Some editorial improvements as well.

Fixes #136.

### CODE REVIEW

File: content/glossary.md [hunk 1]
```diff
@@ -34,88 +35,81 @@ A tree of spans. A trace has a root span that encapsulates all the spans from st
...
-3. **Never sample:**  
-	
-	- Here you’ve asked that the system never traces any request.  
+**Sampling**
+
+Sampling determines how often requests will be traced.
+There are three types of sampling made available from OpenCensus libraries:  
+
+* **Always sample:** With this sampling, every single request is traced.  
+* **Probabilisticly sample:** A probability is set (e.g 0.0001) and the libraries will sample only 1 in 10000 requests.
```
Comment (songy23) [inline content/glossary.md]: nit: consider changing to "A probability is set (e.g 0.0001) and the libraries will sample according to that probability (e.g 1 in 10000 requests)."

File: content/glossary.md [hunk 2]
```diff
@@ -34,88 +35,81 @@ A tree of spans. A trace has a root span that encapsulates all the spans from st
...
+OpenCensus doesn't export each collected measurement. It preaggregates measurements in the process.
+OpenCensus provides the following aggregations:  
 
-    - Characterizing measures as a sum of increasing quantiles e.g number of requests overtime always increases and is quantified by a natural number.  
-	i.e. >= 1  
-  
-2. **Distribution Aggregation:**  
+* **Count:** Reports the count of the recorded data points. For example, number of requests.
+* **Distribution:** Reports the histogram distribution of the recorded measurements.
+* **Mean:** Reports the mean of the recorded measurements.
```
Comment (songy23) [inline content/glossary.md]: I think `Mean` is removed/deprecated?

File: content/glossary.md [hunk 3]
```diff
@@ -8,12 +8,14 @@ date = "2018-05-14T16:11:08-05:00"
...
-Tracking the entire path of execution of a service or a function from start until end. e.g. During an upload, tracking when a file handle is received, opened, sanitized and uploaded to the cloud.  
+
+Distributed Tracing tracks the progression of a request as it is handled by the services and processes that make up a distributed system. Each step in the Trace is called a Span. Each Span contains metadata, such as the length of time spent in the step, and optionally annotations about the work being performed. This information can be used for debugging and performance tuning of distributed systems.
 
 ---
 
 **Span**  
-The unit of traversal during tracing. Each span might have a children spans within it or it might be the first in its generation and called a root.  
+
+A unit work in a trace. Each span might have a children spans within it or it might be the first in its generation and called a root span.
```
Comment (songy23) [inline content/glossary.md]: s/a children spans/children spans

Comment (songy23) [reply content/glossary.md]: Also, this sentence is still confusing because root span can have children spans too. Maybe rephrase as "Each span might have children spans within it. If a span does not have a parent, it is the first in its generation and called a root span."?

File: content/glossary.md [hunk 4]
```diff
@@ -34,88 +35,82 @@ A tree of spans. A trace has a root span that encapsulates all the spans from st
...
+**Sampling**
 
----
+Sampling determines how often requests will be traced.
+There are three types of sampling made available from OpenCensus libraries:  
 
-**Measure**  
-Any quantifiable measurement/metric that you would like to track, it could be number of requests, number of failed responses, bytes downloaded, cache misses, number of radios purchsed etc.  
+* **Always sample:** With this sampling, every single request is traced.  
+* **Probabilisticly sample:** A probability is set (e.g 0.0001) and the libraries will sample only 1 in 10000 requests.
```
Comment (songy23) [inline content/glossary.md]: nit: consider "A probability is set (e.g 0.0001) and the libraries will sample according to that probability (e.g 1 in 10000 requests)."

File: content/glossary.md [hunk 5]
```diff
@@ -34,88 +35,82 @@ A tree of spans. A trace has a root span that encapsulates all the spans from st
...
-    - Characterizing measures over the mean of all of the values and maintains the mean value.  
-  
-4. **Sum Aggregation:**  
+OpenCensus doesn't export each collected measurement. It preaggregates measurements in the process.
+OpenCensus provides the following aggregations:  
 
-    - Characterizing measures over the total of values and maintains the value as a sum.  
+* **Count:** Reports the count of the recorded data points. For example, number of requests.
+* **Distribution:** Reports the histogram distribution of the recorded measurements.
+* **Mean:** Reports the mean of the recorded measurements.
```
Comment (songy23) [inline content/glossary.md]: I think `Mean` is now removed/deprecated?

File: content/glossary.md [hunk 6]
```diff
@@ -34,88 +35,82 @@ A tree of spans. A trace has a root span that encapsulates all the spans from st
...
 
 ---
 
-**Tags**  
-Tags are key-value pairs that are used to store information about metrics for example;  
+**Tags**
 
-+ ip=10.32.103.12  
+Tags are key-value pairs that can be recorded with measurements.
+They later used to breakdown the collected data in the metric collection backend.
```
Comment (songy23) [inline content/glossary.md]: s/They later used to/They are later used to

File: content/glossary.md (hunk 1 above)
Comment (rakyll) [reply content/glossary.md]: Done.

File: content/glossary.md (hunk 2 above)
Comment (rakyll) [reply content/glossary.md]: True, removed.

File: content/glossary.md (hunk 3 above)
Comment (rakyll) [reply content/glossary.md]: Done.

File: content/glossary.md (hunk 6 above)
Comment (rakyll) [reply content/glossary.md]: Done.

**Comment by rakyll** [comment]
All addressed, PTAL.
